lib/index.js

```diff
@@ -24,6 +25,17 @@ module.exports = tools.makeStringTransform('sassify', {
     options.outFile = opts.file;
     options.sourceMapEmbed = true;
     options.sourceMapContents = true;
+    options.importer = function(url, prev) {
+        var importAbsolutePath = path.resolve(path.dirname(prev), url);
+        var underscoredImportAbsolutePath = path.join(path.dirname(importAbsolutePath), '_' + path.basename(importAbsolutePath));
+        self.emit('file', importAbsolutePath);
+        self.emit('file', importAbsolutePath + '.scss');
+        self.emit('file', underscoredImportAbsolutePath);
+        self.emit('file', underscoredImportAbsolutePath + '.scss');
```

Comment:
it's a bit aggressive to do every possible file, but the alternative is `fs.exists`. Not sure which is more/less intrusive?